tests/testthat/test-coerce-from.R

```diff
@@ -13,6 +12,4 @@ test_that("Coercion from other date classes into messydt works", {
   expect_equal(as.Date(as_messydate("2010-10-10"), random), date)
   expect_error(as.POSIXct(as_messydate("-2012"), min))
   expect_error(as.POSIXlt(as_messydate("-2012"), min))
-  skip_on_os("linux")
-  expect_equal(as.character(as.Date(negative, min)), "-028-01-01")
```

Comment:
So we just don't test for this now?

Reply:
Yes, that is correct, we test negative returns for all sytems (i.e. ´lengths()´) but not their printing.